### DISCUSSION TITLE
I wanted to know if there is a way to display errors in TS

### DISCUSSION THREAD

**Discussion by odevnicolas** [zed-industries/zed#6671] [Feature Requests]
I wanted to know if there is a way to display the typo errors, I did a project using TS, but zed didn't alert when a mandatory item was missing in an object for example

<img width="671" alt="Screenshot 2024-01-02 at 16 09 47" src="https://github.com/zed-industries/community/assets/125375345/558b6649-1dad-417a-b46b-6dd03d50397a">
<img width="588" alt="Screenshot 2024-01-02 at 16 10 02" src="https://github.com/zed-industries/community/assets/125375345/44f462b6-2f69-43a3-8916-a6d4997ac0a9">

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
